Cubical/Categories/DistLatticeSheaf/ComparisonLemma.agda

```diff
@@ -0,0 +1,445 @@
+       limOut FLimCone (v , is-trans _ _ _ y≥v x≥y) ⋆⟨ C ⟩ α .N-ob v ∎
+   diagArrow : C [ RanOb limitC i F x , RanOb limitC i G y ]
+   diagArrow = limArrow GYLimCone _ diagCone
+
+
+
+ DLRanFun : Functor (FUNCTOR Bᵒᵖ C) (FUNCTOR Lᵒᵖ C)
+ F-ob DLRanFun = DLRan
+ N-ob (F-hom DLRanFun α) x = limOfArrows (FLimCone α _) (GLimCone α _) (↓nt α x)
+ N-hom (F-hom DLRanFun {x = F} {y = G} α) {x = x} {y = y} x≥y =
+   sym (limArrowUnique (GLimCone α y) _ (diagCone α x y x≥y) _ isConeMorL)
+     ∙ (limArrowUnique (GLimCone α y) _ _ _ isConeMorR)
+   where
+   l = limArrow (FLimCone α y) _ (RanCone limitC i F x≥y)
+           ⋆⟨ C ⟩ limOfArrows (FLimCone α _) (GLimCone α _) (↓nt α y)
+
+   isConeMorL : isConeMor (diagCone α x y x≥y) (limCone (GLimCone α y)) l
+   isConeMorL (u , y≥u) =
+       l ⋆⟨ C ⟩ (limOut (GLimCone α y) (u , y≥u))
+     ≡⟨ ⋆Assoc C _ _ _ ⟩
+       limArrow (FLimCone α y) _ (RanCone limitC i F x≥y)
+         ⋆⟨ C ⟩ (limOfArrows (FLimCone α _) (GLimCone α _) (↓nt α y)
+                   ⋆⟨ C ⟩ (limOut (GLimCone α y) (u , y≥u)))
+     ≡⟨ cong (seq' C (limArrow (FLimCone α y) _ (RanCone limitC i F x≥y)))
+             (limOfArrowsOut (FLimCone α _) (GLimCone α _) _ _) ⟩
+       limArrow (FLimCone α y) _ (RanCone limitC i F x≥y)
+         ⋆⟨ C ⟩ (limOut (FLimCone α _) (u , y≥u) ⋆⟨ C ⟩ α .N-ob u)
+     ≡⟨ sym (⋆Assoc C _ _ _) ⟩
+       (limArrow (FLimCone α y) _ (RanCone limitC i F x≥y)
+         ⋆⟨ C ⟩ (limOut (FLimCone α _) (u , y≥u))) ⋆⟨ C ⟩ α .N-ob u
+     ≡⟨ cong (λ x → x ⋆⟨ C ⟩ (α .N-ob u)) (limArrowCommutes (FLimCone α _) _ _ _) ⟩
+       limOut (FLimCone α x) (u , is-trans _ _ _ y≥u x≥y) ⋆⟨ C ⟩ α .N-ob u ∎
+
+   r = limOfArrows (FLimCone α _) (GLimCone α _) (↓nt α x)
+           ⋆⟨ C ⟩ limArrow (GLimCone α y) _ (RanCone limitC i G x≥y)
+
+   isConeMorR : isConeMor (diagCone α x y x≥y) (limCone (GLimCone α y)) r
+   isConeMorR (u , y≥u) =
+       r ⋆⟨ C ⟩ (limOut (GLimCone α y) (u , y≥u))
+     ≡⟨ ⋆Assoc C _ _ _ ⟩
+       limOfArrows (FLimCone α _) (GLimCone α _) (↓nt α x)
+         ⋆⟨ C ⟩ (limArrow (GLimCone α y) _ (RanCone limitC i G x≥y)
+                 ⋆⟨ C ⟩ (limOut (GLimCone α y) (u , y≥u)))
+     ≡⟨ cong (seq' C (limOfArrows (FLimCone α _) (GLimCone α _) (↓nt α x)))
+          (limArrowCommutes (GLimCone α _) _ _ _) ⟩
+       limOfArrows (FLimCone α _) (GLimCone α _) (↓nt α x)
+         ⋆⟨ C ⟩ limOut (GLimCone α x) (u , is-trans _ _ _ y≥u x≥y)
+     ≡⟨ limOfArrowsOut (FLimCone α x) (GLimCone α x) _ _ ⟩
+       limOut (FLimCone α x) (u , is-trans _ _ _ y≥u x≥y) ⋆⟨ C ⟩ α .N-ob u ∎
+
+
+ F-id DLRanFun {x = F} = makeNatTransPath
+                           (funExt λ x → limOfArrowsId (FLimCone (idTrans F) x))
+ F-seq DLRanFun α β = makeNatTransPath
+                        (funExt λ x → limOfArrowsSeq (FLimCone α x) (GLimCone α x)
+                                                     (GLimCone β x) (↓nt α x) (↓nt β x))
+
+
+
+ --extension of sheaves as functor
+ extSh : Functor ShB ShL
```

Comment:
Maybe expand "Sh" to "Sheaf"? Probably applies to other names as well. This will be more consistent with the naming we have for "Presheaf"